### DISCUSSION TITLE
Size of API token.

### DISCUSSION THREAD

**Discussion by Uttam-EWW** [adonisjs/core#3494] [Help]
## Is size of token will be fixed for every token generated?

**Comment by carlmathisen** [comment]
Not entirely sure what you're asking. The token value is converted to a sha256 hash, which is always 64 characters.

**Reply by Uttam-EWW** [reply]
I'm asking length of token before hashing.

**Reply by thetutlage** [reply]
https://github.com/adonisjs/auth/blob/develop/src/Guards/Oat/index.ts#L60

**Reply by Uttam-EWW** [reply]
Okay, so base token length will be 60 which will be appended to base64 of id of token in in database. 
`<base64 of id>.<token>`
